specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-10-01-preview/containerInstance.json

```diff
@@ -0,0 +1,977 @@
+          "type": "number",
+          "format": "double"
+        }
+      }
+    },
+    "ResourceLimits": {
+      "description": "The resource limits.",
+      "type": "object",
+      "properties": {
+        "memoryInGB": {
+          "description": "The memory limit in GB of this container instance.",
+          "type": "number",
+          "format": "double"
+        },
+        "cpu": {
+          "description": "The CPU limit of this container instance.",
+          "type": "number",
+          "format": "double"
+        }
+      }
+    },
+    "AzureFileVolume": {
+      "description": "The properties of the Azure File volume. Azure File shares are mounted as volumes.",
+      "type": "object",
+      "required": [
+        "shareName",
+        "storageAccountName"
+      ],
+      "properties": {
+        "shareName": {
+          "description": "The name of the Azure File share to be mounted as a volume.",
+          "type": "string"
+        },
+        "readOnly": {
+          "description": "The flag indicating whether the Azure File shared mounted as a volume is read-only.",
+          "type": "boolean"
+        },
+        "storageAccountName": {
+          "description": "The name of the storage account that contains the Azure File share.",
+          "type": "string"
+        },
+        "storageAccountKey": {
+          "description": "The storage account access key used to access the Azure File share.",
+          "type": "string"
+        }
+      }
+    },
+    "EmptyDirVolume": {
+      "description": "The empty directory volume.",
+      "type": "object",
+      "properties": {}
+    },
+    "Volume": {
+      "description": "The properties of the volume.",
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "description": "The name of the volume.",
+          "type": "string"
+        },
+        "azureFile": {
+          "description": "The name of the Azure File volume.",
+          "$ref": "#/definitions/AzureFileVolume"
+        },
+        "emptyDir": {
+          "description": "The empty directory volume.",
+          "$ref": "#/definitions/EmptyDirVolume"
+        }
+      }
+    },
+    "VolumeMount": {
+      "description": "The properties of the volume mount.",
+      "type": "object",
+      "required": [
+        "name",
+        "mountPath"
+      ],
+      "properties": {
+        "name": {
+          "description": "The name of the volume mount.",
+          "type": "string"
+        },
+        "mountPath": {
+          "description": "The path within the container where the volume should be mounted. Must not contain colon (:).",
+          "type": "string"
+        },
+        "readOnly": {
+          "description": "The flag indicating whether the volume mount is read-only.",
+          "type": "boolean"
+        }
+      }
+    },
+    "ContainerGroup": {
```

Comment:
> "ContainerGroup": { [](start = 3, length = 20)

Is this a tracked resource?

Reply:
Yes.

Reply:
Take a [look at this error](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/287783930#L605)

Reply:
I see. This is a known issue. We haven't had a fix for it yet. But we do have a work item tracking it.